src/components/Global/TokenAmountInput/index.tsx

```
@@ -27,64 +26,141 @@
+            switch (displayMode) {
+                case 'STABLE':
+                    tokenValue = value
+                    break
+                case 'TOKEN':
+                    if (!selectedTokenData?.price) throw new Error('Invalid selected token data')
+                    tokenValue = _isInputUsd ? (Number(value) / selectedTokenData.price).toString() : value
+                    break
+                case 'FIAT':
+                    if (!currency?.price) throw new Error('Invalid currency')
+                    tokenValue = _isInputUsd ? value : (Number(value) / currency.price).toString()
+                    const currencyValue = _isInputUsd ? (Number(value) * currency.price).toString() : value
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)